app/app.yaml

```diff
@@ -0,0 +1,39 @@
+components:
+  schemas:
+    App:
+      description: >-
+        A container for emulated protocol and traffic configuration.
+      type: object
+      properties:
+        tcp:
+          description: >-
+            The properties of TCP and its children, 
+          type: array
+          items:
+            $ref: './tcp/tcp.yaml#/components/schemas/App.Tcp'  
+          x-field-uid: 1
+        http:
+          description: >-
+            The properties of HTTP and its children, 
```

Comment:
Is this just HTTP or also HTTPS?  If it is also HTTPS then update comment to "HTTP(S)".